gprofiler/main.py

```diff
@@ -160,7 +162,7 @@ def __init__(
             lambda: RbSpyProfiler(self._frequency, self._duration, self._stop_event, self._temp_storage_dir.name),
             "ruby",
         )
-        if include_container_names:
+        if include_container_names and legacy_profile_api_version is None:
```

Comment:
```suggestion
        if include_container_names and legacy_profile_api_version != "v1":
```

like the other check in this file?